nx/lib/nx/lin_alg.ex

```diff
@@ -1178,6 +1178,10 @@ defmodule Nx.LinAlg do
     * `:max_iter` - `integer`. Defaults to `100`
       Number of maximum iterations before stopping the decomposition
 
+    * `:full_matrices?` - `boolean`. Defaults to `true`
+      If True, u and vt are of shape (M, M), (N, N). Otherwise,
```

Comment:
```suggestion
      If `true`, `u` and `vt` are of shape (M, M), (N, N). Otherwise,
```

Reply:
I would actually default this to `false` if we can, because it generates less output memory footprint and less calculations overall.
@josevalim this would be a breaking change. Thoughts?

Reply:
The default for numpy is `true`, so I would consider mirroring that (and nothing breaks).